help/en/html/tools/Logix.shtml

```diff
@@ -305,7 +320,7 @@
       <p>As mentioned above, Conditionals are statements of the
       form:</p>
 
-      <p><code>if (antecedent) then (consequent).</code></p>
+      <blockquote><code>if (antecedent) then (consequent).</code></blockquote>
```

Comment:
Note that ```<blockquote>``` elements have a specific semantic meaning in HTML. If you wish to indent the code without the semantic meaning of a quote, use a div with a left margin style.

Note also that a web browser *does not* have to indent a <blockquote>, and may override any styling we apply to it.